### ISSUE TITLE
Workspace chats not appearing for workspace members who are already logged in

### ISSUE THREAD

**Issue by roryabraham** [Expensify/App#11451]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Create two accounts, `workspace-owner` and `workspace-employee`.
1. Sign into both accounts in two chrome windows (one incognito)
1. As the `workspace-owner`, create a workspace.
1. Invite `workspace-employee` to the workspace.

## Expected Result:
The `#announce` room and the workspace chat should appear for the `workspace-employee`.

## Actual Result:
The `workspace-employee` must log out and back in before their workspace chat will appear or be discoverable in the the search page.

## Workaround:
Log out and back in

## Platform:
Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 
**Reproducible in staging?:** 
**Reproducible in production?:** 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:** 
**Issue reported by:** 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by trjExpensify** [comment]
Thanks (again!), Rory! I can't seem to reproduce this for the `workspace chat` on staging, it appeared immediately. The issue seems isolated to the `#announce` room not appearing. Did I follow the reproduction steps correctly? 

Gif: http://g.recordit.co/k6kUj0LlS3.gif
Employee view screenie: 

<img width="1511" alt="image" src="https://user-images.githubusercontent.com/16232057/193274354-c075772e-6278-4651-9fee-b8ef2d898a38.png">

**Comment by trjExpensify** [comment]
Bump @roryabraham on the steps to reproduce this: https://github.com/Expensify/App/issues/11451#issuecomment-1263540806

**Comment by JmillsExpensify** [comment]
Since you can't reproduce why don't we go ahead and close?

**Comment by trjExpensify** [comment]
Yeah, I just tried again and the workspace chat appeared immediately. The #announce room required a refresh, not a sign-out/in. Looks like we have an issue for that: https://github.com/Expensify/App/issues/12027

Feel free to reopen Rory, if I missed something on the workspace chats.

**Comment by roryabraham** [comment]
It might've just been due to unreliable pusher events on dev that caused this issue during my local testing 🤷🏼
